lib/utils.js

```diff
@@ -383,9 +388,11 @@ export function moveElement(
   // nearest collision.
   let sorted = sortLayoutItems(layout, compactType);
   const movingUp =
-    compactType === "vertical" && typeof y === "number" ? oldY >= y
-    : compactType === "horizontal" && typeof x === "number" ? oldX >= x
-    : false;
+    compactType === "vertical" && typeof y === "number"
```

Comment:
These kinds of changes should be in a lint PR not bundled with other changes. This makes the commit history & review process more confusing.

Reply:
@STRML these changes are produced by https://github.com/STRML/react-grid-layout/blob/master/package.json#L16. I think we can't skip them here (or we should change precommit hooks/prettier rules)

Reply:
Okay, I'll run prettier on `master` to eliminate this diff.